Formula/tezos-accuser-PtNairob.rb

```diff
@@ -0,0 +1,118 @@
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+class TezosAccuserPtnairob < Formula
```

Comment:
As for the removal of limanet's, there should also be a new `Formula/tezos-node-nairobinet.rb` now